.github/workflows/deploy.yml

```
@@ -0,0 +1,49 @@
+      - master
+    paths: 
+      - "apps/archive/**"
+
+jobs: 
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)